### PR TITLE
Reorganize and de-duplicate skills content

### PR DESCRIPTION
**Pull request by karolk91**
**CLAUDE.md** (217 → 169 lines)
  - Removed duplicated formatting/linting commands from "Test Commands" — `/format` skill is now the single source
  - Collapsed "Development Workflow" from 4 redundant steps to 3, referencing `/format`
  - Moved 40 lines of code review guidelines into the `/review` skill — CLAUDE.md now just points there
  - Renamed "Notes" → "Operational Limits & Requirements"
  - Renamed "Claude Preferences" → "Agent Rules"
  - Fixed "BaBE" typo → "BABE"

  **review/SKILL.md** (35 → 59 lines)
  - Now declared as single source of truth for review criteria
  - Absorbed detailed guidelines from CLAUDE.md: Rust code quality, FRAME pallet standards, security, testing, PR standards
  - Normalized list item style to consistent directive form

  **format/SKILL.md** (1 line)
  - Fixed platform-specific rustup command to platform-agnostic `rustup component add rustfmt --toolchain nightly`

  **build-ui/SKILL.md** (1 line)
  - Added context: "Build the Bulletin Chain web dashboard"

### CODE REVIEW

File: .claude/skills/build-ui/SKILL.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: build-ui
 description: Build the console UI (same as CI)
 ---
 
-Run the console UI build exactly as CI does. From the `console-ui/` directory:
+Build the Bulletin Chain web dashboard (`console-ui/`) exactly as CI does. From the `console-ui/` directory:
 
 1. `npm ci`
 2. `npx papi generate` — skip this step if `$ARGUMENTS` contains `--skip-papi`
```
Comment (bkontur) [inline .claude/skills/build-ui/SKILL.md]: After merge, we will need to update this for naren-client where some mdbook stuff is added

File: .claude/skills/review/SKILL.md [hunk 2]
```diff
@@ -1,33 +1,57 @@
...
-   - Test coverage for new code
-   - Edge cases handled
+5. **Testing**
+   - All new functionality requires unit tests
+   - Test boundary conditions, error paths, and malicious inputs
+   - Complex features need integration tests using `sp-io::TestExternalities`
+   - Features affecting weights need benchmark tests
+
+6. **PR Standards**
+   - Single responsibility: each PR addresses one concern
```
Comment (x3c41a) [inline .claude/skills/review/SKILL.md]: absolutely love it! though, rarely see it 🙃

File: .claude/skills/review/SKILL.md [hunk 3]
```diff
@@ -1,33 +1,57 @@
...
+   - All CI checks pass (`cargo test`, `cargo clippy`, `cargo fmt`)
```
[the remark this answers is not in the document]
Comment (karolk91) [reply .claude/skills/review/SKILL.md]: It is enforced - but I just moved what already existed from `CLAUDE.md` to specific `SKILL.md`
